editor/src/components/canvas/canvas-strategies/canvas-strategies.tsx

```diff
@@ -152,6 +153,16 @@ const keyboardShortcutStrategies: MetaCanvasStrategy = (
   )
 }
 
+const staticReparentStrategies: MetaCanvasStrategy = (
+  canvasState: InteractionCanvasState,
+  interactionSession: InteractionSession | null,
+): Array<CanvasStrategy> => {
+  return mapDropNulls(
+    (factory) => factory(canvasState, interactionSession),
+    [testStaticReparentStrategy],
```

Comment:
Is it likely for there to be more values here? If not I'd just call `testStaticReparentStrategy` and default the result.

Reply:
I'll remove this in the next PR anyway so I don't think there's much use in polishing this